src/server_manager/web_app/build_action.sh

```diff
@@ -25,5 +25,5 @@ do_action server_manager/web_app/build_install_script
 # So, we use a relative path as input to webpack.
 pushd "${ROOT_DIR}" > /dev/null
 # Notice that we forward the build environment if defined.
-webpack --config=src/server_manager/electron_renderer.webpack.js ${BUILD_ENV:+--mode=${BUILD_ENV}}
+yarn webpack --config=src/server_manager/electron_renderer.webpack.js ${BUILD_ENV:+--mode=${BUILD_ENV}}
```

Comment:
with just `webpack`, you're required to have `webpack-cli` installed globally. this uses the local `webpack` bin installed to your `node_modules`. I've done the same with `npx` in the client

It's also additionally problematic to use the global webpack because my globally installed version of webpack might not be the one we support

Reply:
I commented on the other PR. With `yarn do` all the dependencies are in the path. This is not needed, unless you are calling the script directly, which is not supported.